presto-spi/src/main/java/com/facebook/presto/spi/relation/RowExpressionNodeInliner.java

```diff
@@ -11,9 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.sql.relational;
-
-import com.facebook.presto.spi.relation.RowExpression;
+package com.facebook.presto.spi.relation;
```

Comment:
It seems like only the package name is changed but the file is still  in presto-main? Probably we should move all these relation algebra  related things into a standalone package other than presto-spi. cc @highker @rongrong

Reply:
@hellium01 

> It seems like only the package name is changed but the file is still in presto-main?

I think this is the limitation of the GitHub UI. I see the file moved to presto-spi.

Reply:
```
mbasmanova-mbp1:presto mbasmanova$ git show HEAD --name-only
commit 0f5d47872f3ae849d13b7a10c2da10a096d62d6c (HEAD -> move-inliner, origin/move-inliner)
Author: Masha Basmanova <mbasmanova@fb.com>
Date:   Wed Jul 3 10:42:51 2019 -0400

    Move RowExpressionNodeInliner to SPI

presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java
presto-main/src/main/java/com/facebook/presto/sql/planner/RowExpressionVariableInliner.java
presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PickTableLayout.java
presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/SymbolMapper.java
presto-main/src/test/java/com/facebook/presto/sql/planner/TestRowExpressionRewriter.java
presto-spi/src/main/java/com/facebook/presto/spi/relation/RowExpressionNodeInliner.java
presto-spi/src/main/java/com/facebook/presto/spi/relation/RowExpressionRewriter.java
presto-spi/src/main/java/com/facebook/presto/spi/relation/RowExpressionTreeRewriter.java
```

Reply:
Ah, sorry didn't check carefully :) I think the change is quite straightforward, but it's better to have opinions from others as well since it touches SPI :).